src/pycropml/code2nbk.py

```diff
@@ -48,24 +48,25 @@ def generate_nb(self, language, tg_rep, namep, mc_name=None):
         var = ["Auxiliary", "Rate", "State", "Exogenous"]
         if language in ("cs", "java"):
             for v in var:
-                fileVar = Path(os.path.join(tg_rep, "%s%s.%s" % (mc_name.capitalize() if language=="java" else mc_name, v, language)))
+                fileVar = Path(os.path.join(tg_rep, f"{mc_name}{v}.{language}"))
```

Comment:
Add capitalize for java:

   ```
_name = mc_name.capitalize() if language=="java" else mc_name
fileVar = Path(os.path.join(tg_rep, f"{mc_name}{v}.{language}"))
```

Reply:
Thanks for the answer.
This is surely an old fix that may be also fixed somewhere else.

Reply:
Ok. I hope so. Else, I will fix it when any problem appears. :-)